src/app-pages.js

```diff
@@ -12,10 +12,11 @@
  */
 (function (global, undefined) {
 	'use strict';
-	
+
 	const pageModels = {};
 	const pageInstances = {};
 	const activeRoutes = {};
+	const loadedPages = {};
```

Comment:
The loadedPages should be in their own page instance.

Reply:
We also need a way to gc on unused pages.

Reply:
1) Not sure I understand what you mean by that. Each page instance should have a `loadedPages` cache object?

2) How do we determine if a page is unused?

Reply:
1. each loaded page has a framework page instance defined in `pageInstances` the loaded dom should be attached to the instance.

2. it's not the current url. and is only an opt-in